lib/style/pipes.ex

```diff
@@ -204,7 +204,7 @@ defmodule Styler.Style.Pipes do
   # most of these values were lifted directly from credoa's pipe_chain_start.ex
   @literal ~w(__block__ __aliases__ unquote)a
   @value_constructors ~w(% %{} .. <<>> @ {} ^ & fn from)a
-  @infix_ops ~w(++ -- && || in - * + / > < <= >= ==)a
+  @infix_ops ~w(++ -- && || in - * + / > < <= >= == and or != !==)a
```

Comment:
mind adding `===` while you're at it? your `!==` made me think of that

Reply:
Yes, you're right about that. I will add this on the next PR.